src/Sarif.UnitTests/SarifVersionOneToCurrentTests.cs

```diff
@@ -710,5 +693,141 @@ public void SarifTransformerTests_ToCurrent_OneRunWithNotificationsButNoInvocati
 
             v2LogText.Should().Be(v2LogExpectedText);
         }
+
+        [Fact]
+        public void SarifTransformerTests_ToCurrent_NotificationExceptionWithStack()
+        {
+            string v1LogText =
+              @"{
+                  ""version"": ""1.0.0"",
+                  ""runs"": [
+                    {
+                      ""configurationNotifications"": [
```

Comment:
Since this PR is just about stacks and stack frames, let's omit `configurationNotifications`.

Reply:
Done